drm/ResourceManager.cpp

```diff
@@ -66,6 +66,36 @@ static int find_virtio_gpu_card(ResourceManager *res_man, char* path_pattern, in
   }
 }
 
+void ResourceManager::ReloadNode() {
+  char path_pattern[PROPERTY_VALUE_MAX];
+  // Could be a valid path or it can have at the end of it the wildcard %
+  // which means that it will try open all devices until an error is met.
+  int path_len = property_get("vendor.hwc.drm.device", path_pattern,
+                              "/dev/dri/card%");
+  int node_num = 0;
+  path_pattern[path_len - 1] = '\0';
+  for (int idx = 0;; ++idx) {
+    std::ostringstream path;
+    path << path_pattern << idx;
+
+    struct stat buf {};
+    if (stat(path.str().c_str(), &buf) != 0)
+      break;
+
+    node_num++;
+  }
+  if (node_num > card_num) {
```

Comment:
currently we assume the new added card is ivshmem device, can we call ioctl to double confirm that.
Also need add more description for these code.

Reply:
The hwc use the card, while the minigbm open the renderD, i am not sure whether in hwc the card could query the device info like renderD in minigbm.
@phreer , do you know that?

Reply:
Check the node is ivshm device logic is addede.
Also i add the reloaded_ to make sure android only open one ivshm device.